pkgs/servers/emby/default.nix

```diff
@@ -27,18 +27,18 @@ stdenv.mkDerivation rec {
   '';
 
   installPhase = ''
-    mkdir -p $out/bin
-    cp -r * $out/bin
+    mkdir -p "$out/bin"
+    cp -r * "$out/bin"
 
     makeWrapper "${mono54}/bin/mono" $out/bin/MediaBrowser.Server.Mono \
       --add-flags "$out/bin/MediaBrowser.Server.Mono.exe -ffmpeg ${ffmpeg}/bin/ffmpeg -ffprobe ${ffmpeg}/bin/ffprobe"
   '';
 
   meta = {
     description = "MediaBrowser - Bring together your videos, music, photos, and live television";
-    homepage = https://emby.media/;
-    license = stdenv.lib.licenses.gpl2;
+    homepage    = https://emby.media/;
```

Comment:
Please don't align on `=`

Reply:
I've had other contributors actually request me to do this. (I now also prefer it)
Since there still isn't really a formal style guide, I don't think this should be a problem.

Reply:
Done.